### ISSUE TITLE
Long code blocks in markdown forces pages to become too wide

### ISSUE THREAD

**Issue by Jezzamonn** [ludumdare/ludumdare#582]
Example: https://ldjam.com/users/jezzamon/

or to reproduce, enter the following:

    ```
    This text is just something arbitrary and long such that it's longer than the width of the page. The text will be on all one line, which it should be because it's meant to be code but it also mucks up the page layout.
    ```

Ideally we should do something like github or stackoverflow does, and create a horizontal scroll bar (the text above functions as an example of that too)

**Comment by povrazor** [comment]
If somebody wants to look in to why this isn't working as expected, please do.

The original styles are coming from:

`/src/custom/prism/prism.css`

But the actual fix should probably go here:

`/src/com/content-common/common-body-markup.less`

**Comment by povrazor** [comment]
When I was playing with it, I found I could "fix it" by setting the width of the `<pre>` to a fixed value. The problem is the width of the pre needs to be dynamic (i.e. fit the parent). On mobile it gets squished.

**Comment by ishanpm** [comment]
Yay for flexbox!

**Comment by Jezzamonn** [comment]
nice. I tried to look at this by setting `position: absolute` on the `pre` tags, but that runs into other issues.
I don't like css. :(

<marquee direction="down" width="250" height="200" behavior="alternate" style="border:solid">
  <marquee behavior="alternate">
    bring back styling in html
  </marquee>
</marquee>

**Comment by imerr** [comment]
Setting `overflow: hidden;` on `#content` seems to fix it

Not sure if that'd break anything else though

**Comment by povrazor** [comment]
Thanks for looking in to this everyone! I've committed a fix based on @imerr's `overflow` suggestion. It should be fixed now. Let me know if you run in to any other problems. 

The fix for the nav buttons was to change their `display` style to `inline block`, as before they were `inline` (they're actually `<a>` links).

@ishanpm's solution of flexboxes could have worked too, as I am using flexboxes elsewhere on the site. Correcting the `display` type for the buttons was just what I tried first.
